### ISSUE TITLE
Pie chart with single value falls back to default color

### ISSUE THREAD

**Issue by Amli** [DmitryBaranovskiy/g.raphael#24]
using this code:
    <div id="holder"></div>
    <script>
        var r = Raphael("holder");
        r.g.piechart(320, 240, 100, [50, 50], {colors: ['00ffff']});
    </script>
the fill color should be #00ffff

fix should be on line 23 of g.pie.js
    fill: opts.colors && opts.colors[0] || this.g.colors[0] || "#666",
instead of 
    fill: this.g.colors[0],

**Comment by rokcarl** [comment]
Hmmm, this still doesn't work for me. I still get the empty (white?) circle with a 1 px black border.
Also, no legend appears and rollovers don't work.

**Comment by scottwb** [comment]
Agreed - I made this fix in: http://github.com/scottwb/g.raphael/commit/369b360de5058cad0a3d94cf5ba5678791c3f7c1

@rokcarl - I think you might also need the fix for Issue 3, which I've seen others fix, but I also fixed in http://github.com/scottwb/g.raphael/commit/c0e44b68291108682563d10f8ca60aad34b73200

**Comment by ilikepi** [comment]
It would be nice if this fix could be pushed, given that it's literally one line...

**Comment by markstory** [comment]
This fixed worked well for me too, thanks :)

**Comment by barbitoff** [comment]
I`ve faced the same problem, and have found the same fix for it
